scripts/packaging/publishupmpackages.ps1

```diff
@@ -0,0 +1,48 @@
+<#
+.SYNOPSIS
+    Publishes the Mixed Reality Toolkit Unity Package Manager (UPM) packacges.
+.DESCRIPTION
+    Publishes UPM packages for the Mixed Reality Toolkit.
+.PARAMETER PackageDirectory
+    Where should we find the packages to upload? Defaults to ".\artifacts"
+.PARAMETER ProjectRoot
+    The root folder of the project.
+#>
+param(
+    [string]$PackageDirectory = ".\artifacts\upm",
+    [string]$ProjectRoot
+)
+
+$startPath = "$(Get-Location)"
+
+if (-not $ProjectRoot) {
+    # ProjectRoot was not specified, presume the current location is Root\scripts\packaging
+    $ProjectRoot = Resolve-Path "$startPath\..\.." 
+}
+$ProjectRoot = Resolve-Path -Path $ProjectRoot
+Write-Output "Project root: $ProjectRoot"
+
+$PackageDirectory = Resolve-Path -Path $PackageDirectory
+Write-Output "Package directory: $PackageDirectory"
+
+$npmCommand = "npm"
+
+$cmdFullPath = "$env:systemroot\system32\cmd.exe"

```

Comment:
Can `npm` just be run directly from this powershell script instead of launching `cmd`?

Reply:
it wasnt able to when the script was installing node on demand. it can likely be changed back now

Reply:
changed